api/src/test/java/org/pmiops/workbench/billing/FreeTierBillingServiceTest.java

```diff
@@ -852,7 +852,6 @@ private DbUser createUser(String email) {
     return userDao.save(new DbUser().setUsername(email));
   }
 
-  // we only alert/record for BillingMigrationStatus.NEW workspaces
```

Comment:
no longer used as a distinction as of #6922 - instead we use a deletion date cutoff, which enables us to do much less work